### ISSUE TITLE
Questions about the output loss

### ISSUE THREAD

**Issue by yyyyqy** [zhykoties/DeepAR#15]
Hi, here is my problem: 
What is the physical significance of the horizontal and vertical coordinates of the output loss image？

**Comment by zhykoties** [comment]
If you are referring to https://github.com/zhykoties/TimeSeries/blob/master/experiments/base_model/figures/elect_loss_summary.png,

horizontal axis is the number of training steps, and vertical axis is the value of the loss at that step.
